### DISCUSSION TITLE
wmbusmeters on Raspberry Pi OS (Debian 12)

### DISCUSSION THREAD

**Discussion by Tom1000** [wmbusmeters/wmbusmeters#1289] [Show and tell]
Hello wmbus community!

I'd like to share my experience with wmbusmeters on a Raspi 4B running Raspberry Pi OS (Debian 12) and a METIS-II AMBER (now Würth Electronics) AMB8626-M USB stick. In general this went quite smoothly.  I used the latest Experimental build available in the Debian repositories, which is a bit outdated unfortunately (v1.13, ~1 year old). wmbusmeters ran and detected the stick without problems. 

Running wmbusmeters as a service however required the user "wmbusmeters" to be added to the tty and/or dialout group (+ restart afterwards).

Thanks to all who are maintaining & contributing to this project!

**Comment by volkerdeHaas** [comment]
HI, I am desperately trying to get the service running after boot in my raspi. But it still does not work. I added wmbusmeters to tty group and rebooted but still the same. The service is loaded but inactive. I always need to start it manually

**Reply by Tom1000** [reply]
Did you also add wmbusmeters to dialout group and restarted afterwards?

**Reply by volkerdeHaas** [reply]
Thanks for your reply. But I've asked in different threads and someone reminded me, that I need to manually activate a demon in Linux to finally install it as active. I'm not a Linux expert, so this was a dummy fault. After having it activated, it starts smoothly after reboot and my system is running perfectly now
